deepfence_server/reporters/completion/completion.go

```diff
@@ -43,14 +42,25 @@ func FieldValueCompletion[T reporters.Cypherable](ctx context.Context, req Compl
 	}
 	defer tx.Close()
 
-	query := `
-	MATCH (n:` + dummy.NodeType() + `)
-	WHERE n.` + req.FieldName + ` =~ '^` + req.Completion + `.*'
-	RETURN DISTINCT n.` + req.FieldName
+	query := ""
+
+	if req.ScanID != "" {
+		query = `
+		MATCH (n{node_id: $scan_id}) -[:DETECTED]-> (m) -[:IS]-> (r:` + dummy.NodeType() + `) ` +
+			getWhereClause("r", req.FieldName, req.Completion) +
```

Comment:
so I need to add one more check there in case of empty `completion`, right?

though if we don't put that check it might still work (` WHERE r.cve_type =~ '^.*'`)

Reply:
No empty completion is handled already as "everything"
If you look below, empty completion will have the regexp: `^.*`

Reply:
got it!

Reply:
done